### ISSUE TITLE
Accessibility checks seem to fail sometimes (flaky)

### ISSUE THREAD

**Issue by bramdebouvere** [Adyen/adyen-web#3399]
**Describe the bug**
In our pipeline, we run automated accessibility tests over our web applications. When we go over the payments, they seem to randomly fail at times. At other times, there seem to be no issues. It almost seems as if Adyen is loading different versions of fields at times.

EDIT: is it possible that the aria roles are being added after the initial render of the component? In that case, there might be a race condition between lighthouse checking the page and adding the aria roles.

**To Reproduce**
1. Scan the Adyen component using Chrome Lighthouse

**Expected behavior**
The scans complete without issues

**Screenshots**
<img width="803" height="585" alt="Image" src="https://github.com/user-attachments/assets/50702b1d-2481-48b5-bcfd-debc75187a3a" />

**Desktop (please complete the following information):**
 - OS: Linux / Windows
 - Browser Chrome (Automated Lighthouse Flows)
 - Version "@adyen/adyen-web": "~6.14.0",

**Smartphone (please complete the following information):**
 - not tested

**Additional context**
After upgrading to v~6.14.0, this issue did not pop up at first. It was only a few weeks later that we randomly started getting errors.

**Comment by m1aw** [comment]
Hey @bramdebouvere 

We run some a11y tests on our [mystoredemo.io](https://www.mystoredemo.io/#/) and we couldn't reproduce your issue on lighthouse.

Also it's unclear from the screenshot above what are the missing `aria-` attributes for the `role="radio"` because `aria-checked` is there, so is the parent `radiogroup` role. 

Can you share some more details that can help us reproduce this issue?

**Comment by bramdebouvere** [comment]
Hi @m1aw ,

Sorry for the late response.

Apparently, some things are not rendered in their final state when the adyen component is rendered on screen for the first time. The aria roles seem to be added later.
I now did a workaround by waiting for the class `adyen-checkout__dropin--ready` to appear.

```
await page.waitForSelector('& >>> enviso-adyen-payment >>> .adyen-checkout__dropin--ready');
```

When I wait for that specific class, the component seems to be initialized completely, and the aria roles seem to be applied.
